pkg/manifests/manifests.go

```diff
@@ -351,11 +353,47 @@ func MachineConfig(component string) (*machineconfigv1.MachineConfig, error) {
 }
 
 func getIgnitionConfig() ([]byte, error) {
+	var files []igntypes.File
+
 	// load SELinux policy
-	seLinuxPolicyContent := base64.StdEncoding.EncodeToString(rteassets.SELinuxPolicy)
+	files = addFileToIgnitionConfig(files, rteassets.SELinuxPolicy, 644, seLinuxRTEPolicyDst)
+
+	// load RTE notifier OCI hook config
+	notifierHookConfigContent, err := getTemplateContent(rteassets.HookConfigRTENotifier, map[string]string{
+		templateNotifierBinaryDst: filepath.Join(defaultScriptsDir, "rte-notifier.sh"),
+	})
+	if err != nil {
+		return nil, err
+	}
+	files = addFileToIgnitionConfig(
+		files,
+		notifierHookConfigContent,
+		644,
+		filepath.Join(defaultOCIHooksDir, "rte-notifier.json"),
+	)
+
+	// load RTE notifier script
+	notifierScript, err := getTemplateContent(rteassets.RTENotifierScript, map[string]string{
+		// TODO: we should use the variable passed from the deployer or operator
```

Comment:
yes, this should be a parameter from the deployer that is passed through up to this code

Reply:
For now, I prefer to hardcore it because otherwise, we can not guarantee that the code will work on top of the OpenShift, because of the SELinux. For the hardcoded path we know what SELinux permissions grant to the container.

Reply:
Works for me, but let's document this in a code comment.